addon/components/cfb-form-editor/general.js

```diff
@@ -20,11 +21,40 @@ export default Component.extend(ComponentQueryManager, {
 
   notification: service(),
   intl: service(),
+  calumaOptions: service(),
 
-  didReceiveAttrs() {
+  /**
+   * Adds an uneditable prefix to the input field.
+   * This uses manual DOM manipulation to avoid adding a single-use component.
+   */
+  addSlug() {
+    const input = this.element.querySelector('[name="slug"]');
+
+    if (
+      this.namespace &&
+      input &&
+      !input.classList.contains("slugnamespace-input")
+    ) {
+      const span = document.createElement("span");
+      const parent = input.parentElement;
+
+      Object.assign(span, {
+        className: "slugnamespace-slug",
+        innerHTML: `${this.namespace}-`
+      });
+      parent.classList.add("slugnamespace");
+      parent.insertBefore(span, input);
+    }
+  },
```

Comment:
Is this _exactly_ the same as for questions? If yes, it might be worth extracting this into a shared function?

Reply:
It is. But I'd rather concentrate my efforts on writing the custom component.

Reply:
Alright!